src/app/nation/nation.component.scss

```diff
@@ -0,0 +1,15 @@
+#admin_name {
```

Comment:
This is very repetitive, and more fitting as a class in the main styles.scss since this is reusable.